sdk/python/tests/test_cli_local.py

```diff
@@ -71,3 +71,42 @@ def test_workflow() -> None:
 
         result = runner.run(["teardown"], cwd=repo_path)
         assert result.returncode == 0
+
+
+def test_non_local_feature_repo() -> None:
```

Comment:
probably missing something obvious here, but what's the "non-local" part about this test?

Reply:
It just means you can do something like `FeatureStore(repo_path="very/far/away/repo/")` and then have a `feature_store.yaml` that has a relative path like `registry: data/registry.db`. It will then build a registry path that is `very/far/away/repo/data/registry.db`

Reply:
oh got it, thanks